mod-source-record-manager-server/src/main/java/org/folio/dao/JournalRecordDaoImpl.java

```diff
@@ -129,6 +130,25 @@
       .onFailure(e -> LOGGER.error("Error saving JournalRecord entity", e));
   }
 
+  @Override
+  public Future<List<RowSet<Row>>> saveBatch(JsonArray entities, String tenantId) {
+    Promise<List<RowSet<Row>>> promise = Promise.promise();
+    try {
+      String query = format(INSERT_SQL, convertToPsqlStandard(tenantId), JOURNAL_RECORDS_TABLE);
+      List<Tuple> tupleList = new ArrayList<>();
+      for (int i = 0; i < entities.size(); i++) {
```

Comment:
As for me its better to make this conversion in the service layer, and DAO should accept just List<JornalRecord> journalRecords

Reply:
@SerhiiNosko Done, thanks!